samples/drivers/vbatt/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _vbatt_sample:
+
+Battery Voltage Measurement Sample
+##################################
+
+Overview
+********
+
+This sample demonstrates how one can use the configuration of the Zephyr ADC infrastructure to
+measure the voltage of the device power supply using a voltage-divider. The sample assumes a
+regulator that provides a fixed voltage to IO38 of the esp32 behind a voltage divider. In this
+sample the voltage divider consists of 300k and 100k Ohms resistors. It will then print the measured
+battery voltage to the console. If you take care to not power the device thru the usb port, you can
+see how the battery voltage decreases over time, Thus indicating the current charging level.
```

Comment:
I'm not sure what you mean by "indicates the current charging level".  Your sample only prints the battery voltage.  Without knowing the battery capacity and/or empty voltage, I'm not sure how one infers the charge level.